static/app/components/charts/chartZoom.tsx

```diff
@@ -344,6 +344,7 @@ class ChartZoom extends Component<Props> {
         utc,
         start,
         end,
+        isGroupedByDate: true,
```

Comment:
@nikkikapadia this is the prop that was messing up the x-axes. Since you _need_ this to enable chartzoom, I decided to always pass it along through the props. This way, any chart that has a `ChartZoom` wrapper should be a timeseries chart and get this by default even if the zooming component is disabled.